pkg/ddl/000038_oidc_user.down.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+DROP INDEX email_unique_idx;
+ALTER TABLE auth_users
+DROP COLUMN IF EXISTS email,
+    DROP COLUMN IF EXISTS encrypted_password;
```

Comment:
Doesn't this kill all users on `migrate down`?

Reply:
oops! made a mess here